LICENSE.md

```diff
@@ -52,39 +52,36 @@ The User accepts the following liability:
  - This Mod may alter fundamental parts of the Game.
  - The User is liable for all damages resulting from the use or misuse of the Mod.
 
-3. Right to use
+3. Right to use and examine
 --------
 
 The User is allowed to install the Mod and play without restriction.  
 The User may not limit access to the Mod in any way, except as provided by the Game.  
 
-4. Right to examine
---------
-
-The User may decompile compiled binaries and examine the resulting source code.  
-The User may examine the source code as it exists in public repositories.
+The User may decompile compiled binaries.  
+The User may examine this Mod's source code.
 
-5. Right to distribute
+4. Right to distribute
 --------
 
 The Author reserves the right to distribute the Mod.  
 The Author may designate a person or organization as a Distributor.  
 A Distributor is granted the right to distribute the Mod.  
+When distributing binaries, a Distributor must provide means to obtain the Mod's source code at no charge, for any User who recieves a copy of this Mod from the Distributor.  
```

Comment:
This isn't necessary. I'd be ok with adding a "The Author may require that a Distributor provide a direct link to the Codebase", but requiring that a distributor also distribute source is outside the scope of what the distribution section attempts to address.